content/root/quick-start-guide.textile

```diff
@@ -15,65 +17,84 @@ languages:
+* Include an Ably Client Library SDK in a project.
+* Establish a realtime connection to Ably.
+* Subscribe to a channel to receive messages.
+* Publish a message on a channel.
+* Close a connection to Ably.
 
-h2(#adding-library). Adding the Ably library
+Ably Client Library SDKs are available in multiple programming languages. Some SDKs have a realtime and a REST interface, whilst others only have a REST interface. The "SDK downloads":https://ably.com/download page lists availability. For further information on when it's best to use the realtime or REST SDK, see "the best practice guide":https://ably.com/documentation/best-practice-guide#choosing-between-realtime-rest-mqtt.
 
-This quickstart guide is in <span lang="javascript">JavaScript</span><span lang="nodejs">Node.js</span><span lang="java">Java</span><span lang="ruby">Ruby</span><span lang="python">Python</span><span lang="php">PHP</span><span lang="objc">Objective-C</span><span lang="swift">Swift</span><span lang="csharp">C# .Net</span><span lang="flutter">Flutter</span> If you would like to see this guide in another language, please select your preferred language from the language selector nav above.
+p(tip#terms). The terms used in the guide such as 'connection', 'channel', 'message', 'publish' and 'subscribe' are described in the "glossary":/glossary.
+
+You're currently viewing the guide in <span lang="javascript">JavaScript</span><span lang="nodejs">Node.js</span><span lang="java">Java</span><span lang="ruby">Ruby</span><span lang="objc">Objective-C</span><span lang="python">Python</span><span lang="php">PHP</span><span lang="swift">Swift</span><span lang="csharp">C# .Net</span><span lang="flutter">Flutter</span>. If you would like to view it in another language, use the selector above to change it. 
+
+h2(#step-1). Step 1: Add the Ably Client Library SDK
 
 blang[javascript].
-  When using Ably Realtime or REST in a browser, simply include the Ably JavaScript library within the @<head>@ of your HTML page as follows:
+  The Ably Client Library SDK is available via CDN.
+
+  To get started with your project, reference the SDK within the @<head>@ of an HTML page:
 
-  bc[html]. <script src="//cdn.ably.io/lib/ably.min-1.js"></script>
+  bc[html]. <script src="//cdn.ably.com/lib/ably.min-1.js"></script>
 
-  You can also obtain the library as an "NPM module":https://www.npmjs.com/package/ably.
+  The SDK is also available as an "NPM module":https://www.npmjs.com/package/ably.
 
 blang[nodejs].
-  The Ably Realtime and REST library is obtainable as an "NPM module":https://www.npmjs.com/package/ably. Simply add to your @package.json@ file or install manually:
+  The Ably Client Library SDK is available as an "NPM module":https://www.npmjs.com/package/ably.
+
+  To get started with your project, add the SDK into your @package.json@ file or install it manually:
```

Comment:
I'm not sure if, for either npm modules or ruby gems, that anyone would choose to add these to package.json/Gemfile manually?

Reply:
This is a good point! I've removed that part of the sentence for both.